tutorxqueue/plugin.py

```diff
@@ -146,21 +146,19 @@ def grade_submission(
     print_result(
         context,
         "grade_submission",
-        (
-            submission_id,
-            submission_key,
-            grade,
-            correct,
-            message,
-        ),
+        submission_id,
+        submission_key,
+        grade,
+        correct,
+        message,
```

Comment:
Is this change necessary due to xqueue changes or Py compatibility?

Reply:
@DawoudSheraz This is because of Python compatibility. I had problems calling the [print_result](https://github.com/overhangio/tutor-xqueue/pull/30/files#diff-116762e6266c82183b0eac2bb932f6e0bffa337e78068d91459f93907dfe576dR146) function so I fixed the *args and **kwargs typing. Please check this [link](https://adamj.eu/tech/2021/05/11/python-type-hints-args-and-kwargs/) to get further details.